dev/build/windows/makecoq_mingw.sh

```diff
@@ -1043,19 +1092,38 @@ function make_coq {
   make_ocaml
   make_lablgtk
   make_camlp5
-  if 
+  if
     case $COQ_VERSION in
-      git-*) build_prep https://github.com/coq/coq/archive ${COQ_VERSION##git-} zip 1 coq-${COQ_VERSION} ;;
-      *)     build_prep https://coq.inria.fr/distrib/V$COQ_VERSION/files coq-$COQ_VERSION tar.gz ;;
+      # e.g. git-v8.6 => download from https://github.com/coq/coq/archive/v8.6.zip
+      # e.g. git-trunk => download from https://github.com/coq/coq/archive/trunk.zip
+      git-*) 
+        COQ_BUILD_PATH=/build/coq-${COQ_VERSION}
+        build_prep https://github.com/coq/coq/archive ${COQ_VERSION##git-} zip 1 coq-${COQ_VERSION}
+        ;;
+      
+      # e.g. /cygdrive/d/coqgit
+      /*)
```

Comment:
I think you should also check for relative paths here, `./*)`.  (I'm trying to set up CI testing, and don't have easy access to the absolute path before cygwin is installed.)